### ISSUE TITLE
Translated string issues

### ISSUE THREAD

**Issue by willmot** [humanmade/backupwordpress#926]
Looking through our [originals file](https://github.com/humanmade/backupwordpress/blob/master/languages/backupwordpress.pot), there are a few issues.

- [ ] Unnecesary HTML in strings. _Where we are wrapping a placeholder in html we should instead include that html in the placeholder so it isn't shown to translators. For example: https://github.com/humanmade/backupwordpress/blob/master/languages/backupwordpress.pot#L703_
- [ ] Some urls that shouldn't be marked as translatable. _Url's which have different language versions available should be marked as translatable, for example [here](https://github.com/humanmade/backupwordpress/blob/master/languages/backupwordpress.pot#L1056), however if the url is only available in one language then there's no point, like [here](https://github.com/humanmade/backupwordpress/blob/master/languages/backupwordpress.pot#L219)
- [ ] There's probably no point marking internal only error messages for translation, e.g. https://github.com/humanmade/backupwordpress/blob/master/languages/backupwordpress.pot#L711

**Comment by pdewouters** [comment]
Yes, I noticed a few issues, I was going to do at the same time as https://github.com/humanmade/backupwordpress/issues/897

**Comment by willmot** [comment]
Related to https://github.com/humanmade/backupwordpress/issues/1065

**Comment by willmot** [comment]
This may well now be out of date.
